content/client-lib-development-guide/features.textile

```diff
@@ -274,10 +276,11 @@ h3(#realtime-connection). Connection
 ** @(RTN2b)@ @echo@ should be @true@ by default; @false@ will prevent messages published by the client being echoed back
 ** @(RTN2d)@ @clientId@ contains the provided @clientId@ option of @ClientOptions@, unless @clientId@ is @null@
 ** @(RTN2e)@ Depending on the authentication scheme, either @accessToken@ contains the token string, or @key@ contains the API key
-** @(RTN2f)@ API version param @v@ should be @0.8@
+** @(RTN2f)@ API version param @v@ should be @0.9@
 * @(RTN3)@ If connection option @autoConnect@ is true, a connection is initiated immediately; otherwise a connection is only initiated following an explicit call to @connect()@
 * @(RTN4)@ EventEmitter and states:
 ** @(RTN4a)@ Implements @EventEmitter@ and emits events for state changes @INITIALIZED@, @CONNECTING@, @CONNECTED@, @DISCONNECTED@, @SUSPENDED@, @CLOSING@, @CLOSED@, @FAILED@
+** @(RTN4g)@ Additionally, an @ERROR@ event is emitted that contains an @ErrorInfo@ object with details on an error that has occurred for the @Connection@
```

Comment:
I did review this and think it sounded weird, but it's just a copy of http://docs.ably.io/client-lib-development-guide/features/#RTL2c that is currently in the spec.  I will reword as this spec item should simply state that that an `ERROR` event can be emitted, not that it will.  So it's a generic spec stating that a channel / connection has the ability to emit errors, that's it.